bin/setup

```diff
@@ -10,33 +10,25 @@ def system!(*args)
   system(*args) || abort("\n== Command #{args} failed ==")
 end
 
-Dir.chdir APP_ROOT do
+chdir APP_ROOT do
   # This script is a starting point to setup your application.
   # Add necessary setup steps to this file.
 
-  puts "\n== Enabling default add-ons =="
```

Comment:
why are some of these removed from the setup binary?

Reply:
same response as above